### PR TITLE
Fix a crash when response payload is null

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

**Comment by TroubleMakerBen** [comment]
Just one question: did you change the provisioning profile on purpose?

**Comment by jaelim-ms** [comment]
Yeah.. I refreshed the provisioning profiles in developer account as well as bitrise.

File: MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m [hunk 1]
```diff
@@ -267,92 +267,100 @@ - (void)checkLatestRelease:(NSString *)updateToken releaseHash:(NSString *)relea
...
+
+             // Error instance for JSON parsing.
+             NSError *jsonError = nil;
+
+             // Success.
+             if (statusCode == MSHTTPCodesNo200OK) {
+               MSReleaseDetails *details = nil;
+               if (data) {
+                 id dictionary =
+                     [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
```
Comment (clpolet) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: Looks like this can produce an error, we should handle it.

Comment (clpolet) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: This will be part of another task as discussed off line.

**Comment by TroubleMakerBen** [comment]
Let's merge this to develop. I need to update the changelog.

**Pull request merged**